src/algebra/algebra/basic.lean

```diff
@@ -1468,6 +1470,37 @@ by rw [←(one_smul A m), ←smul_assoc, algebra.smul_def, mul_one, one_smul]
 @[simp] lemma algebra_map_smul (r : R) (m : M) : ((algebra_map R A) r) • m = r • m :=
 (algebra_compatible_smul A r m).symm
 
+variables (S A)
+
+@[simp] lemma algebra_map_algebra_map (r : R) : algebra_map S A (algebra_map R S r) =
+  algebra_map R A r :=
+by rw [algebra.algebra_map_eq_smul_one, algebra_map_smul, algebra.algebra_map_eq_smul_one]
+
+@[simp] lemma algebra_map_comp_algebra_map : (algebra_map S A).comp (algebra_map R S) =
+  algebra_map R A :=
```

Comment:
We have this already:

https://github.com/leanprover-community/mathlib/blob/5b75f5ac2a1897f514941f95665f83480959026b/src/algebra/algebra/tower.lean#L92-L93

Reply:
I deleted `algebra_map_comp_algebra_map`